src/components/ContentBlocks/BlockRichText/BlockRichText.mock.ts

```diff
@@ -0,0 +1,9 @@
+import { loremIpsum } from 'lorem-ipsum';
+
+export const RICH_TEXT_MOCK = `
+  # Title
+
+  ${loremIpsum({ count: 20 })}
```

Comment:
voeg nog een lijst van items toe in html en markdown want hier was vroeger een probleem mee dat het niet correct renderde

```
  * one	
  * two	
  * three	
  <ul>	
		<li>1</li>	
		<li>2</li>	
		<li>3</li>	
  </ul>
```